doc/user/content/license.md

```diff
@@ -1,14 +1,44 @@
 ---
-title: "License"
-description: "Human-readable license for Materialize"
+title: "License and privacy policy"
```

Comment:
Lol -- don't know why it didn't actually register that I could just modify this page to mirror the emulator section.

https://preview.materialize.com/materialize/31865/self-managed/v25.1/license/